.github/workflows/qcom-preflight-checks-reusable-workflow.yml

```diff
@@ -121,9 +115,19 @@ jobs:
     steps:
       - name: 'Checkout Repository'
         uses: actions/checkout@v4
+      - name: Set base ref and head ref
+        id: setrefs
+        run: |
+          if [ "${{github.event_name}}" == "push" ]; then
+            echo "base_ref=${{ github.event.before }}" >> $GITHUB_OUTPUT
+            echo "head_ref=${{ github.sha }}" >> $GITHUB_OUTPUT
+          elif [ "${{ github.event_name }}" == "pull_request_target" ]; then
+            echo "base_ref=${{ github.event.pull_request.base.sha }}" >> $GITHUB_OUTPUT
+            echo "head_ref=${{ github.event.pull_request.head.sha }}" >> $GITHUB_OUTPUT
+          fi
```

Comment:
What if the event is not `push` or `pull_request_target`? Should we have a sensible fallback/default in case someone updates their qcom-preflight `on:` conditions unexpectedly? Or should we skip dependency-review if either of these values are missing?